src/lib/plugin.js

```diff
@@ -127,6 +127,12 @@ class FiveBellsLedger extends EventEmitter2 {
     const ledgerMetadata = yield this._fetchLedgerMetadata()
     this.accountUriTemplate = ledgerMetadata.urls.account
 
+    // Set ILP prefix
+    this.prefix = ledgerMetadata.ilp_prefix || this.fallbackPrefix
+    if (!this.prefix) {
+      throw new Error('Unable to set prefix from ledger or from local config')
+    }
```

Comment:
Yeah, I definitely think that the connector should be able to override the ledger's self-reported prefix. Maybe log a warning, but for sure we want to allow it.